### DISCUSSION TITLE
graphql and prisma

### DISCUSSION THREAD

**Discussion by halindraprakoso** [prisma/prisma#12481] [Q&A] [answered]
Hello  👋,

I am exploring graphql with prisma and I have a question about resolvers. 
I'm noticing that all the example that I have found simply use `context`, for instance:
```js
context.prisma.user.findmany()
```
However, it seems to fetch all the data from the DB and just do some filtering before sending it to the client, right? I assume it might be problematic if someone has a large dataset.

Then I googled and found this video: https://egghead.io/lessons/javascript-prevent-graphql-queries-from-over-fetching-from-the-database

The question is: is it common to supply `info` to the prisma client so that it does not overfetch like that example from the video?

**Comment by austincrim** [comment, marked as answer]
Hey @halindraprakoso 👋 ,

Using the `info` parameter is definitely possible and beneficial in some cases! It really all depends on your setup and the resolver you are concerned with.

Where have you been learning about GraphQL and Prisma?

**Reply by halindraprakoso** [reply]
Thanks for the answer Austin! 

I'm just learning by exploring example repos. But I've learned some more about this from prisma's youtube video https://www.youtube.com/watch?v=7oMfBGEdwsc&t=1108s

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
